tests/foreman/ui/test_computeresource_ec2.py

```diff
@@ -31,6 +31,8 @@
 
 if not setting_is_set('ec2'):
     skip('skipping tests due to missing ec2 settings', allow_module_level=True)
+if not setting_is_set('http_proxy'):
+    skip('skipping tests due to missing http_proxy settings', allow_module_level=True)
```

Comment:
I think this should only skip tests that actually require proxy

Reply:
@lhellebr this is required because I have updated ui tests to use proxy.

Reply:
Yes, and I think only the test that requires the proxy should be skipped - not all the tests in the file.

Reply:
ahh, right. got it. :P

Reply:
@lhellebr updated the pr. Thanks for noticing this.